### DISCUSSION TITLE
How can I get build output using fly?

### DISCUSSION THREAD

**Discussion by maestrow** [concourse/concourse#7481] [Help & Support] [answered]
Is there a way to get build output using command line?

**Comment by taylorsilva** [comment, marked as answer]
Yes!
```
fly watch                      Stream a build's output (aliases: w)
```

And when you `fly trigger-job` you can use `--watch` to watch it.

**Reply by maestrow** [reply]
Yeah, I'm aware about watch mode. But my question is about how to get logs from finished builds (not just current running)? I can see them in UI, but what about cli?

**Reply by taylorsilva** [reply]
You can also use `fly watch` to see the output of a finished build as well! :)

**Reply by maestrow** [reply]
Oh, thank you very much :). It's just doesn't seems obvious for me :).
